src/www/base.webpack.js

```diff
@@ -47,6 +48,14 @@ exports.makeConfig = (options) => {
       extensions: ['.tsx', '.ts', '.js'],
       fallback: {'url': require.resolve('url/')},
     },
+    optimization: {
+      minimizer: [new TerserPlugin({
+        terserOptions: {
+          keep_classnames: true,
+          keep_fnames: true,
```

Comment:
Why is this needed?

Reply:
Our logging code relies on `Class.prototype.name` so we need to preserve function names in addition to class names. See the [terser docs](https://github.com/terser/terser#minify-options) for more information.